keyboards/ares/keymaps/default/keymap.c

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 Maarten Dekkers <maartenwut@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define _MA 0
+#define _FN 1
+
+#define ______ KC_TRNS
```

Comment:
If you use the 7 underscore variant, you don't need to define this as its already included in QMK_KEYBOARD_H

Reply:
I would very much like to see merlin's suggestion followed here.

Reply:
Agreed, please use the standard `_______` alias.
```suggestion
```
Also remove the extra blank line below this.